apps/test/util/setupGoogleBlocklyGlobal.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Provides the basic frame for running Google Blockly.  In particular, this
+ * will create a basic dom, load googleblockly.js and put the contents into
+ * the global space as global.Blockly.
+ */
+
+export default function setGoogleBlocklyGlobal() {
+  // Initialize browser environment.
+  document.body.innerHTML = '<div id="codeApp"><div id="app"></div></div>';
```

Comment:
Do we need this `codeApp` div? I don't think we use this specific ID in lab2, but is blockly looking for it?